README.md

```diff
@@ -331,6 +332,12 @@ If you want to contribute to this list (please do), send me a pull request.
 * [gqlr](https://github.com/schloerke/gqlr) - GraphQL server package for R.
 * [ghql](https://github.com/ropensci/ghql) - GraphQL client package for R.
 
+<a name="lib-julia" />
+
+### Julia Libraries
+
+* [Diana.jl](https://github.com/codeneomatrix/Diana.jl) - GraphQL Julia implementation.
```

Comment:
After looking more in depth - when people say "GraphQL implementation" they usually mean a way to create schemas and execute queries with that language. I'd say this is more of a GraphQL client right now, so it would be good to update the description.